cached/cached.go

```diff
@@ -67,7 +70,12 @@ func (c *cachedLookuperPublisher) Get(ctx context.Context, id string, version in
 }
 
 func (c *cachedLookuperPublisher) get(ctx context.Context, id string, version int64) (swarm.Address, error) {
-	cctx, cancel := context.WithTimeout(ctx, c.timeout)
+	timeout := c.timeout
```

Comment:
This is not what we spoke about. We should have a function to SetTimeout and update the value. Init is not related to this package. It is something we do for other reasons in the driver. So this package should not have logic to know anything about init. It should blindly use the timeout it is given.